### PR TITLE
[FOIMOD-3389] add generic cover email template

### PR DESCRIPTION
**Pull request by JieunSon96**
[FOIMOD-3389](https://citz-imb.atlassian.net/browse/FOIMOD-3389)

- Add a generic cover email template with revision.
- Add a generic HTML cover email template on all states.
- Removed unused console log.

### CODE REVIEW

File: forms-flow-web/src/components/FOI/FOIRequest/FOIRequest.js [hunk 1]
```diff
@@ -1027,7 +1027,7 @@ const FOIRequest = React.memo(({ userDetail, openApplicantProfileModal }) => {
     return (
       requestState !== StateEnum.intakeinprogress.name &&
       requestState !== StateEnum.unopened.name &&
-      requestState !== StateEnum.open.name &&
+      //requestState !== StateEnum.open.name &&
       requestState !== StateEnum.appfeeowing.name &&
```
Comment (milosdes) [inline forms-flow-web/src/components/FOI/FOIRequest/FOIRequest.js:1031]: I think we want the communication tab available in all of these states.

File: forms-flow-web/src/components/FOI/customComponents/ContactApplicant/util.tsx [hunk 2]
```diff
@@ -31,7 +30,12 @@ export const getExtensiondetails = (requestExtensions:any, type: string) => {
     return ["","","","","","",""]
 }
 
-export const getExtensionType = (requestExtensions: any) => {
+export const getExtensionType = (requestDetails:any, requestExtensions: any) => {
+
+  if (requestDetails.currentState === "Open") {
+    return "NA";
+  }
```
Comment (milosdes) [inline forms-flow-web/src/components/FOI/customComponents/ContactApplicant/util.tsx:37]: Will we need to do this for the other 3 states (intake in progress, unopened, app fee owing)?

**Review by milosdes — APPROVED** [review]

File: forms-flow-web/src/components/FOI/FOIRequest/FOIRequest.js (hunk 1 above)
Comment (JieunSon96) [reply forms-flow-web/src/components/FOI/FOIRequest/FOIRequest.js:1031]: Yes right! The visibility of this in other states will be handled in a different ticket story, considering any other template options that may be possible based on the state and other criteria. So for now, I made sure that this template is always visible only in states beyond 'Open'.

File: forms-flow-web/src/components/FOI/customComponents/ContactApplicant/util.tsx (hunk 2 above)
Comment (JieunSon96) [reply forms-flow-web/src/components/FOI/customComponents/ContactApplicant/util.tsx:37]: Good point! we need to show the OIPC and PB templates in states beyond 'Open', so it seems like a necessary addition.

**Pull request merged**